### ISSUE TITLE
Error: cannot check server release: cannot find neither /etc/uyuni-release nor /etc/susemanagere-release

### ISSUE THREAD

**Issue by j-kihet** [uyuni-project/uyuni-tools#424]
`mgradm upgrade podman` command ends in an error `Error: cannot check server release: cannot find neither /etc/uyuni-release nor /etc/susemanagere-release`
The server can be revived by going in to the container and running commands to start postgresql and spacewalk-service.
The upgrade seems to have worked but this isn't the correct outcome.
Upgrade was from 2024.05 to 2024.07 on podman
Container tools are up to date, from EL_9 repo.
```
mgradm upgrade podman
12:23PM INF Welcome to mgradm
12:23PM INF Executing command: podman
12:23PM INF Cannot find RPM image for registry.opensuse.org/uyuni/server:latest
12:23PM INF Running podman pull registry.opensuse.org/uyuni/server:latest
Trying to pull registry.opensuse.org/uyuni/server:latest...
Getting image source signatures
Copying blob 82f4a02bb4a5 done   |
Copying blob e29629bb36a2 done   |
Copying blob 490ea0c3696a done   |
Copying config 1ec7fabb91 done   |
Writing manifest to image destination
1ec7fabb91d83438fa8c6750abffbfe752fa30a6fd0fb811491edd8bcb09a1b9
Error: cannot check server release: cannot find neither /etc/uyuni-release nor /etc/susemanagere-release
```

**Comment by aaannz** [comment]
This seems to me a timing issue. I checked released container image and `/etc/uyuni-release` is present. So this seems like check for it is happening too soon and container is not yet completely started. This should be fixed by https://github.com/uyuni-project/uyuni-tools/pull/386

This fix is not yet included in your version (`head 311105e` I see in another issue), but will be in next released uyuni-tools.

**Comment by admd** [comment]
@aaannz also there is typo here /etc/susemanager**e**-release

**Comment by aaannz** [comment]
Indeed, but that's only a logging typo (see [in the code](https://github.com/SUSE/uyuni-tools/blob/Manager-5.0/mgradm/shared/utils/exec.go#L251) and only in SUSE branch.

For uyuni that code was refactored and those strings are no longer in the codebase.
